specification/validated-relationships-service-api.yaml

```diff
@@ -1089,119 +1156,97 @@ components:
+                  description: Denotes that the Bundle is a collection of resources returned as a result of a search.
+                  enum: [searchset]
+     
+    RelatedPersonBundle:
+      allOf:
+        - $ref: "#/components/schemas/Searchset"
+        - type: object          
+          properties:
+            link:
+              type: array
+              items:
+                type: object
+                properties:
+                  relation:
+                    type: string
+                    description: |
+                      Links related to this Bundle - see:
+                      http://www.iana.org/assignments/link-relations/link-relations.xhtml
+                    example: "self"
+                  url:
+                    type: string
+                    format: uri
+                    example: https://api.service.nhs.uk/validated-relationships/FHIR/R4/RelatedPerson?_include=RelatedPerson%3apatient&identifier=https%3A%2F%2Ffhir.nhs.uk%2FId%2Fnhs-number%7C9000000017
+            entry:
+              type: array
+              description: |
+                A collection of resources contained within the Bundle.
 
-            An empty bundle suggests that the proxy does not have patients they can act on behalf of.
-          items:
-            anyOf:
-              - $ref: "#/components/schemas/RelatedPerson"
-              - $ref: "#/components/schemas/Patient"
+                An empty bundle suggests that no candidate proxy/patient links were identified.
+              items:
+                anyOf:
+                  - $ref: "#/components/schemas/RelatedPersonBundleEntry"
+                  - $ref: "#/components/schemas/PatientBundleEntry"
+
+    ConsentBundle:
+      allOf:
+        - $ref: "#/components/schemas/Searchset"
+        - type: object
+          properties:
+            entry:
+              type: array
+              description: |
+                A collection of resources contained within the Bundle.
 
-    RelatedPerson:
+                An empty bundle suggests that the proxy has no proxy relationships.
+              items:
+                anyOf:
+                  - $ref: "#/components/schemas/RelatedPersonBundleEntry"
+                  - $ref: "#/components/schemas/PatientBundleEntry"
+                  - $ref: "#/components/schemas/ConsentBundleEntry"  
+    
+    RelatedPersonBundleEntry: 
       type: object
-      description: The proxy's details. This includes their relationship to the referenced patient.
+      description: A RelatedPerson bundle entry.
       properties:
         fullUrl:
           type: string
           description: The canonical URL of the resource in the entry
           example: "https://api.service.nhs.uk/validated-relationships/FHIR/R4/RelatedPerson/BE974742"
         resource:
+          $ref: "#/components/schemas/RelatedPerson"                        
+        search:
           type: object
           properties:
-            resourceType:
```

Comment:
I'm wondering if an old(er) version has been taken as not sure why these have been removed 
These go a lot for the deleted sections.

Obvs happy to go through it with you when we are both back (w.c. 2nd Sept)

Reply:
Have stepped through with Ameya and happy now with the why and the how